app/models/expression_file_info.rb

```diff
@@ -14,29 +14,43 @@ class ExpressionFileInfo
   UNITS_VALUES = [nil, 'UMI-corrected raw counts', 'raw counts']
   validates :units, inclusion: {in: UNITS_VALUES}
 
-  BIOSAMPLE_INPUT_TYPE_VALUES = [nil, 'whole cell', 'single nuclei', 'bulk']
+  BIOSAMPLE_INPUT_TYPE_VALUES = [nil, 'Whole cell', 'Single nuclei', 'Bulk']
   validates :biosample_input_type, inclusion: {in: BIOSAMPLE_INPUT_TYPE_VALUES}
 
   MULTIMODALITY_VALUES = [nil, 'CITE-seq', 'Patch-seq']
   validates :multimodality, inclusion: {in: MULTIMODALITY_VALUES}
 
   LIBRARY_CONSTRUCTION_VALUES = [nil,
```

Comment:
We ultimately want `LIBRARY_CONSTRUCTION_VALUES` to be the same as library_preparation_protocol from the metadata convention (ie. if we accept values in both places, we would be checking the data matches for data integrity) not sure if we want/need these names to be the same in code right now. For the user facing text in the expression matrix upload wizard, we should definitely use "Library preparation protocol".

Reply:
> We ultimately want `LIBRARY_CONSTRUCTION_VALUES` to be the same as library_preparation_protocol from the metadata convention (ie. if we accept values in both places, we would be checking the data matches for data integrity) not sure if we want/need these names to be the same in code right now. For the user facing text in the expression matrix upload wizard, we should definitely use "Library preparation protocol".

I've made this change on my PR: #776